src/components/EditFreakForm/EditFreakForm.jsx

```diff
@@ -109,24 +134,35 @@ function EditFreakForm({ freak, onChange, onSubmit }) {
 
       <Form.Group className="mb-3">
         <Form.Label>Skills</Form.Label>
-        <Form.Text className="text-muted">
-          <br />
-          Example (Elm, JS, Swift...)
-        </Form.Text>
-        <Form.Control
-          as="textarea"
-          data-testid="skills-input"
-          rows={ 3 }
-          value={ freak.skills }
-          onChange={ triggerChange(FreakModelKeys.skills) }
+        <Select
+          options={ skillOptions }
+          menuPlacement="top"
+          testid="skills-input"
+          isMulti
+          value={ freak.skills.map(mapSkill) }
+          onChange={ handleSelectChange(FreakModelKeys.skills) }
         />
       </Form.Group>
     </Form>
   );
 }
 
+const skillShape = {
+  value: PropTypes.string,
+  name: PropTypes.string,
+};
+
 EditFreakForm.propTypes = {
-  freak: PropTypes.shape(FreakModelDefault),
+  freak: PropTypes.shape({
+    firstName: PropTypes.string,
+    lastName: PropTypes.string,
+    email: PropTypes.string,
+    description: PropTypes.string,
+    role: PropTypes.string,
+    level: PropTypes.string,
+    norm: PropTypes.string,
+    skills: PropTypes.arrayOf(PropTypes.shape(skillShape)),
+  }),
```

Comment:
Extract the `PropType.shape` for the Freak in the Freak model file, and import it from there to use it here.